BEAMS3D/Sources/beams3d_init_vmec.f90

```diff
@@ -330,6 +331,7 @@ SUBROUTINE beams3d_init_vmec
          IF (ier == 0 .and. bphi /= 0) THEN ! We have field data
             ! Save Grid data
             S_ARR(i,j,k) = MAX(sflx,0.0)
+            RHO_ARR(i,j,k) = SQRT(S_ARR(i,j,k))
```

Comment:
In the three places where S_ARR is set from the VMEC file, immediately afterward I set `RHO_ARR(i,j,k) = SQRT(S_ARR(i,j,k))`.

Probably to be cleaner I should just do `RHO_ARR = SQRT(S_ARR)` once at the end.